client/src/components/Tooltips.tsx

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import { InfoIcon, IconProps } from "@chakra-ui/icons";
+import {
+  Text,
+  HStack,
+  VStack,
+  Image,
+  Icon,
+  Tooltip,
+  TooltipProps,
+  ComponentWithAs,
+} from "@chakra-ui/react";
+
+interface AdviceProps {
+  text: string;
+  icon?: ComponentWithAs<"svg", IconProps>;
+}
+
+const AdviceTip: React.FC<TooltipProps> = ({ ...props }) => (
+  <Tooltip
+    bg="white"
+    color="gray.600"
+    border="1px"
+    borderColor="gray.400"
+    shadow="xs"
+    rounded="md"
+    hasArrow
+    arrowSize={10}
+    arrowShadowColor="gray" // NOTE: The "gray.400" style colors don't work in this field.
+    ml="5"
+    mr="5"
```

Comment:
```suggestion
    mx={5}
```
I tend to use `{}` for attributes that really aren't strings.